tests/unit/neptune/new/test_stringify_unsupported.py

```diff
@@ -531,3 +533,20 @@ def test_integers_outside_32bits(self, run):
 
         with assert_no_warnings():
             run["data"] = data
+
+    def test_float_warnings(self, run):
```

Comment:
Let's test it somewhere else, I don't think it's that related to stringify value

Reply:
Moved the tests 😇